CHANGES.md

```diff
@@ -5,10 +5,132 @@ all changes see the [closed
 milestones](https://github.com/cylc/cylc/milestones?state=closed) for each
 release.
 
+-------------------------------------------------------------------------------
+## __cylc-7.8.0 (2018-11-27)__
+
+Minor release with over 120 issues closed. Significant issues include:
+
+### Enhancements
+
+[#2809](https://github.com/cylc/cylc/pull/2809) - __auto-migration__ - suite
+server programs can now be told (via global config) to self-migrate to
+another available host, e.g. for server maintenance. (The pool of cylc hosts 
+should see a shared filestem).
+
+[#2614](https://github.com/cylc/cylc/pull/2614) and 
+[#2821](https://github.com/cylc/cylc/pull/2821) - __web-based job log viewer__ -
+ `cylc review` (migration of "Rose Bush" from the Rose project). 
+
+[#2693](https://github.com/cylc/cylc/pull/2693) - __auto host selection__ -
+`cylc run` and `cylc restart` can now select the best available host based on
+several metrics, on which to launch the suite server program. See `cylc
+get-host-metrics` and global config documentation. (The pool of cylc hosts
+should see a shared filestem).
+
+[#2339](https://github.com/cylc/cylc/pull/2339) - __general external
+triggering__: tasks can trigger off of arbitrary user-defined Python functions
+called periodically by the suite server program, with built-in functions
+for suite-state (inter-suite) triggering and clock triggering (these
+deprecate the existing suite-state polling tasks and clock-triggered tasks).
+
+[#2734](https://github.com/cylc/cylc/pull/2734) - __EmPy templating__
+support, as an alternative to Jinja2. _"EmPy allows embedding plain Python code
+within templates and doesn't enforce any particular templating philosophy."_
+
+[#2733](https://github.com/cylc/cylc/pull/2733) - enhanced Jinja2 support:
+- __import pure Python modules__ in the same way as template modules
+- Jinja2Tests and Jinja2Globals, for custom "is" tests and global variables
+  (c.f. our existing Jinja2Filters for custom filters).
+
+[#2682](https://github.com/cylc/cylc/pull/2682) - new built-in
+Jinja2 filter to convert ISO8601 date-time durations to
+decimal seconds or hours.
+
+[#2842](https://github.com/cylc/cylc/pull/2842) - `cylc gui` and
+`cylc graph` - better integration with system desktop themes, including dark
+themes; and other minor graph visualization improvments. 
+
+[#2807](https://github.com/cylc/cylc/pull/2807) - task output events (event
+handlers can now be triggered when a task reports a registered output message).
+
+[#2868](https://github.com/cylc/cylc/pull/2868) - a new task
+runtime config item `exit-script`, for scripting to be executed at the
+last moment after successful job completion. (Companion of `err-script`).
+
+[#2781](https://github.com/cylc/cylc/pull/2781) and
+[#2854](https://github.com/cylc/cylc/pull/2854) - improved suite
+server program logging.
```

Comment:
May want to mention that `log/suite/err` is gone.

Reply:
Done.